src/pages/de/core-concepts/astro-pages.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Seiten
+description: Eine Einführung in die Astro-Seiten
+i18nReady: true
+---
+
+**Seiten** sind ein spezieller Typ von [Astro-Komponenten](/de/core-concepts/astro-components/), die sich im Unterverzeichnis `src/pages/` befinden. Sie sind verantwortlich für das Routing, das Laden von Daten und das gesamte Seitenlayout für jede HTML-Seite in deiner Website.
+
+### Dateibasiertes Routing
+
+Astro nutzt eine Routing-Strategie, die **dateibasiertes Routing** genannt wird. Jede `.astro`-Datei im `src/pages`-Verzeichnis wird zu einer Seite oder einem Endpunkt auf deiner Website, basierend auf ihrem Dateipfad.
+
+📚 Lies mehr über [Routing in Astro](/de/core-concepts/routing/)
+
+### Seiten-HTML
+
+Astro-Seiten müssen eine vollständige `<html>...</html>` Seitenantwort zurückgeben, einschließlich `<head>` und `<body>`. (`<!doctype html>` ist optional und wird automatisch hinzugefügt.)
+
+```astro
+---
+// Beispiel: src/pages/index.astro
+---
+<html>
+  <head>
+    <title>Meine Homepage</title>
+  </head>
+  <body>
+    <h1>Willkommen auf meiner Webseite!</h1>
+  </body>
+</html>
+```
+
+### Nutzung von Seitenlayouts
+
+Um zu vermeiden, dass sich dieselben HTML-Elemente auf jeder Seite wiederholen, kannst du gemeinsame `<head>`- und `<body>`-Elemente in Ihre eigenen [Layout-Komponenten](/de/core-concepts/layouts/) verschieben. Du kannst so viele oder so wenige Layout-Komponenten verwenden, wie du möchtest.
+
+```astro
+---
+// Beispiel: src/pages/index.astro
+import MySiteLayout from '../layouts/MySiteLayout.astro';
+---
+<MySiteLayout>
+  <p>Mein Seiteninhalt, verpackt in einem Layout!</p>
+</MySiteLayout>
```

Comment:
```suggestion
---
// Beispiel: src/pages/index.astro
import MeinLayout from '../layouts/MeinLayout.astro';
---
<MeinLayout>
  <p>Mein Seiteninhalt, umgeben von einem Layout!</p>
</MeinLayout>
```

Not a mandatory change, but this is a 1:1 copy of my translated version of the layouts page that is referenced here, so if the reader is going there next, they would be able to recognize the same code.

Many of the translated pages are localizing the component and file names to emphasize that these are just arbitrary names given by the developer rather than having a special meaning in Astro. I therefore also tried to use localized names whenever it made sense.